src/main/java/cz/startnet/utils/pgdiff/PgDiff.java

```diff
@@ -272,47 +280,46 @@ private static void updateSchemas(final PrintWriter writer,
             }
 
             PgDiffTriggers.dropTriggers(
-                    writer, oldSchema, newSchema, searchPathHelper);
+                    writer, oldSchema, newSchema, searchPathHelper,arguments);
```

Comment:
I don't think we need to add an arguments parameter here.  There needs to be a better way of sharing config details with all of these objects...